presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -731,8 +731,9 @@ exists as we've rolled back to the previous state.
     -----------+------+-----------+---------
     (0 rows)
 
-Iceberg Connector Limitations
------------------------------
+Table with delete files
+-----------------------
 
-* The ``SELECT`` operations on Iceberg Tables with format version 2 do not read the delete files
-  and remove the deleted rows as of now (:issue:`20492`).
+Iceberg V2 tables support `row-level deletion <https://iceberg.apache.org/spec/#delete-formats>`,
```

Comment:
```suggestion
Iceberg V2 tables support row-level deletion. For more information see `Row-level deletes <https://iceberg.apache.org/spec/#row-level-deletes>`_ in the Iceberg Table Spec.  
```

Links can be tricky. This link did not build locally (see screenshot). 
<img width="889" alt="Screenshot 2023-10-31 at 10 48 50 AM" src="https://github.com/prestodb/presto/assets/7013443/fd9c65ed-6971-4b61-8aa9-b6f3547d3319">

Suggestion here edits the link for a successful local build, and revises the link introduction text following the [GitLab docs style guide for links](https://docs.gitlab.com/ee/development/documentation/styleguide/#links).